### ISSUE TITLE
Recent button stops working after removing launcher

### ISSUE THREAD

**Issue by Knud3** [0x192/Universal-Android-Debloater#245]
Samsung S22 with newest updates and Android 12.

After removing `com.sec.android.app.launcher` recent button (vertical hamburger button from bottom) stops working. After reinstall and reboot it works again.

```
2022-05-31 21:44:32 INFO  [src\core\utils.rs:221] [Advanced] pm uninstall --user 0 com.sec.android.app.launcher -> Success
2022-05-31 21:44:50 INFO  [src\core\utils.rs:221] [Advanced] cmd package install-existing --user 0 com.sec.android.app.launcher -> Package com.sec.android.app.launcher installed for user: 0
```

**Comment by lawson85** [comment]
That's the same with removing any launcher on any device with Android 10+

**Comment by Knud3** [comment]
Good to know. That information would be nice to add to description.

**Comment by 0x192** [comment]
@Knud3 

[Does it fix your issue ?](https://www.droidwin.com/recents-apps-button-not-working-with-custom-launchers/)

**Comment by Knud3** [comment]
@0x192 

No, it did not. I had animations already disabled. Tried again with reboot and all, but recent button action do not work. Thanks for suggestion anyway.

**Comment by chaorace** [comment]
I can confirm the issue on the Z Flip 3 under latest firmware. Recents break with a custom launcher if you disable `com.sec.android.app.launcher`. Disabling all animations does not resolve the issue.

**Comment by Knud3** [comment]
Same thing happens with `com.android.launcher3` in Nokia T20.

**Comment by MenarikanCerita** [comment]
hi, i also just try to disable this app and same thing happen to my device (Samsung M32 4G), so i assume we just can't remove it from samsung devices without any consequences.
PS: before disable the One UI launcher i've installed Olauncher, my plan is to use it as my main launcher and remove the OEM launcher cause i think it's consuming too much source
